### DISCUSSION TITLE
Show rss feeds without registation or login.

### DISCUSSION THREAD

**Discussion by Hrani-zdarma** [FreshRSS/FreshRSS#5395] [Q&A]
Thank for your powerfull script. Is there way to show rss feed without regaistration.

Thank you for answer.

**Comment by Alkarex** [comment]
Hello,
Check the anonymous mode in the settings:

https://freshrss.github.io/FreshRSS/en/admins/13_Default_user.html#anonymous-reading-mode

![image](https://github.com/FreshRSS/FreshRSS/assets/1008324/fa541626-1075-4e4e-b2d0-267bc72e7fc0)
